tensorflow/core/kernels/parameterized_truncated_normal_op.cc

```diff
@@ -190,6 +184,14 @@ struct TruncatedNormalFunctor<CPUDevice, T> {
           while (sample < limit_sample) {
             const auto rand = dist(&gen_copy);
             const int size = rand.size();
+
+            // Vectorized intermediate calculations for uniform rejection
+            // sampling.
+            // Simply allocate Uniform::kResultElementCount for the two arrays
+            // since they are only used by UniformDistribution.
+            Eigen::array<T, Uniform::kResultElementCount> z;
+            Eigen::array<T, Uniform::kResultElementCount> g;
```

Comment:
Can we replace `Uniform::kResultElementCount` with `size`? This way we don't have to worry about index out of bound.
```suggestion
            Eigen::array<T, size> z;
            Eigen::array<T, size> g;
```

Reply:
Done. I keep the comments unchanged for clear explanation.